core/server/models/role.js

```diff
@@ -51,13 +51,15 @@ Role = ghostBookshelf.Model.extend({
         if (_.isNumber(roleModelOrId) || _.isString(roleModelOrId)) {
             // Grab the original args without the first one
             origArgs = _.toArray(arguments).slice(1);
+
             // Get the actual role model
+            // @TODO: test
             return this.findOne({id: roleModelOrId, status: 'all'}).then(function then(foundRoleModel) {
                 // Build up the original args but substitute with actual model
                 var newArgs = [foundRoleModel].concat(origArgs);
 
                 return self.permissible.apply(self, newArgs);
-            }, errors.logAndThrowError);
```

Comment:
I thought about and I don't think it's a behaviour change, but tell me if you still think it is after explaining why.


`errors.logAndThrowError` get's called when `this.findOne` throws an Error.
`errors.logAndThrowError` logs and throws the error (again)

This function call **always** happens in a HTTP request, so logging happens now automatically through - no need to log it twice (https://github.com/kirrg001/Ghost/blob/logging/core/server/middleware/index.js#L88)

And i don't see any reason why to throw the error?